llvm/lib/Passes/StandardInstrumentations.cpp

```diff
@@ -18,6 +18,7 @@
 #include "llvm/Analysis/CallGraphSCCPass.h"
 #include "llvm/Analysis/LazyCallGraph.h"
 #include "llvm/Analysis/LoopInfo.h"
+#include "llvm/CodeGen/StableHashing.h"
```

Comment:
we should pull `StableHashing.h` out of `CodeGen` first

Reply:
https://github.com/llvm/llvm-project/pull/67704, although I'll wait for this PR to go in first